config/master.key

```diff
@@ -0,0 +1 @@
+8f20a0d2f2c77af163fe46190f7ab27e
```

Comment:
I don't know if this would impact anything with the deployed project, but best practices is to not commit this file (it's a secret).

Check this out for more info: https://www.engineyard.com/blog/rails-encrypted-credentials-on-rails-5.2